whylog/tests/tests_config/test_parsers_and_rules_filter.py

```diff
@@ -0,0 +1,102 @@
+import itertools
+import os.path
+from unittest import TestCase
+
+from whylog.config import YamlConfig
+from whylog.tests.utils import ConfigPathFactory
+
+path_test_files = ['whylog', 'tests', 'tests_config', 'test_files']
+
+
+class TestBasic(TestCase):
+    @classmethod
+    def setUpClass(cls):
+        cls.connection_error_line = "2016-04-12 23:54:45 Connection error occurred on comp1. Host name: host1"
+        cls.lost_data_line = "2016-04-12 23:54:43 Data is missing at comp2. Loss = 230 GB. Host name: host2"
+        cls.data_migration_line = "2016-04-12 23:54:40 Data migration from comp1 to comp2 failed. Host name: host2"
+
+        path = os.path.join(*path_test_files)
+        multiple_path = os.path.join(path, 'test_investigation_plan_files')
+        parsers_path, rules_path, log_types_path = ConfigPathFactory.get_path_to_config_files(path)
+        multiple_parsers_path, multiple_rules_path, _ = ConfigPathFactory.get_path_to_config_files(
+            multiple_path
+        )
+
+        cls.simple_config = YamlConfig(parsers_path, rules_path, log_types_path)
+        cls.multiple_parsers_config = YamlConfig(multiple_parsers_path, rules_path, log_types_path)
+        cls.complexed_config = YamlConfig(
+            multiple_parsers_path, multiple_rules_path, log_types_path
+        )
+
+    @classmethod
+    def get_names_of_parsers(cls, parser_list):
+        return sorted([parser.name for parser in parser_list])
```

Comment:
Assign to a temporary variable and call `.sort()` to cut the memory usage by half. It is not important here, it's just so you know.